docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -19,331 +19,362 @@ tags:
 
-Temporal Cloud supports secure programmatic access through API Key Authentication.
+:::
 
-## Overview
+Temporal Cloud API key authentication provides secure access to the Temporal Service from your applications.
+API keys are a common industry standard for simple access control.
+They need only one key to authenticate.
+[mTLS certificates](https://docs.temporal.io/references/configuration#tls) offer higher security but they require more complex setup and configuration.
+API keys simplify deployment and management when mTLS is not needed.
 
-Temporal Cloud API Keys provide identity-based authentication where an identity can be either a Temporal User or a [Service Account](/cloud/service-accounts)
-Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+This document introduces Temporal Cloud's API key features:
 
-The process is as follows:
+- [API key overview](#overview)
+- [Global Administrator API key management](#manage-api-keys)
+- [User API key management](#user-api-keys)
+- [Manage API keys for Service Accounts](#serviceaccount-api-keys)
+- [API keys for Namespace authentication](#namespace-authentication)
+- [Use API keys to authenticate](#using-apikeys)
+- [API key best practices](#best-practices)
+- [Troubleshoot your API key use](#troubleshooting)
+- [API keys: Frequently Asked Questions](#faqs)
 
-```text
-API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
-```
+## API key overview {#overview}
 
-API Keys support 2 main use cases:
+Temporal Cloud API keys offer industry-standard identity-based authentication for Temporal users and [Service Accounts](/cloud/service-accounts).
+Each key is a unique identity.
+It is linked to role-based access control (RBAC) settings to ensure secure and appropriate access.
 
-- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, Namespaces, certificates, and user identities.
-  - API Keys for use in Cloud Operations Tools are in Public Preview
-- _**Namespace Authentication**_: API Keys can be used as the Namespace authentication mechanism for executing and managing workflows from the SDK and Temporal CLI as an alternative to using mTLS-based authentication.
-  - API Keys for use in Namespace Authentication are in Pre Release
+The authentication process follows this pathway:
 
-API Keys are an opt-in feature that must be enabled by a Global Administrator for use. For instructions on API Key Management for your organization, see [Global Administrator API Key Management](#ga-manage-api-keys).
+```text
```

Comment:
Would love to have a diagram here rather than monospace

Reply:
Let's see what I can whip up. I am not going through the design department, though.

Reply:
Lucid chart done. 
![apikeyrbac](https://github.com/user-attachments/assets/205b392f-091a-4d06-b0d7-5940aededd85)